tests/test_defects.py

```diff
@@ -617,23 +563,7 @@ def test_getNumGoodPixels(self):
         badBox = Box2I(Point2I(85, 98), Extent2I(4, 7))
         testImage.mask[badBox] |= BADBIT
 
-        self.assertEqual(imageSize - badBox.getArea(), self.defaultTask._getNumGoodPixels(mi, 'BAD'))
-
-    def test_edgeMasking(self):
-        """Check that the right number of edge pixels are masked by _setEdgeBits()"""
-        testImage = self.flatExp.clone()
-        mi = testImage.maskedImage
-
-        self.assertEqual(countMaskedPixels(mi, 'EDGE'), 0)
-        self.defaultTask._setEdgeBits(mi)
-
-        hEdge = self.defaultConfig.nPixBorderLeftRight
-        vEdge = self.defaultConfig.nPixBorderUpDown
-        xSize, ySize = mi.getDimensions()
-
-        nEdge = xSize*vEdge*2 + ySize*hEdge*2 - hEdge*vEdge*4
-
-        self.assertEqual(countMaskedPixels(mi, 'EDGE'), nEdge)
```

Comment:
Is this test not relevant anymore?

Reply:
I think this was a more extreme case of commented to avoid errors during testing leading to deletion.  Restored as well.